src/onegov/people/models/person.py

```diff
@@ -167,9 +185,24 @@ def vcard_object(self, exclude=None, include_memberships=True):
             line = result.add('url')
             line.value = self.website
 
-        if 'address' not in exclude and self.address:
+        if 'location_address' not in exclude and self.location_address:
```

Comment:
Have you tested this with the various vCard consumers such as outlook etc.? I think there were problems here previosuly, right @BreathingFlesh?

Reply:
I will validate the vcard. I found a tool https://l0b0.wordpress.com/2009/12/25/vcard-parser-and-validator/

Reply:
I brought street address, code and city to one address entry to make it readable on windows.